testnet-node/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '2.2'
```

Comment:
I imagine we probably want:

* A clear prompt in the main README: "If you're looking to develop an integration to run a Core API node, see the [./testnet-node](testnet-node folder)."
* A clear README in this folder explaining that this is intended for development; how to use this docker file, where to find docs for integrations, and where to find docs for configuring a node in production.